src/wp-includes/blocks/index.php

```diff
@@ -40,26 +39,62 @@ function register_core_block_style_handles() {
 		'editorStyle' => 'editor',
 	);
 
+	static $core_blocks_meta;
+	if ( ! $core_blocks_meta ) {
+		$core_blocks_meta = require $includes_path . 'blocks/blocks-json.php';
+	}
+
+	$files          = false;
+	$transient_name = 'wp_core_block_css_files';
 	/*
 	 * Ignore transient cache when the development mode is set to 'core'. Why? To avoid interfering with
 	 * the core developer's workflow.
 	 */
 	if ( ! wp_is_development_mode( 'core' ) ) {
```

Comment:
Maybe put the two calls to this into a variable, like `$can_use_cached`? That avoids the double call and clarifies the intention of the check. That approach is also used in other places in core.
```suggestion
	$transient_name = 'wp_core_block_css_files';

	/*
	 * Ignore transient cache when the development mode is set to 'core'. Why? To avoid interfering with
	 * the core developer's workflow.
	 */
	$can_use_cached = ! wp_is_development_mode( 'core' );
	
	if ( $can_use_cached ) {
```